src/Storages/MergeTree/PartMovesBetweenShardsOrchestrator.cpp

```diff
@@ -194,7 +194,7 @@ void PartMovesBetweenShardsOrchestrator::stepEntry(const Entry & entry, zkutil::
                 /// This wait in background schedule pool is useless. It'd be
                 /// better to have some notification which will call `step`
                 /// function when all replicated will finish. TODO.
-                storage.waitForAllReplicasToProcessLogEntry(log_entry, true);
+                storage.waitForAllReplicasToProcessLogEntry(zookeeper_path, log_entry, 0);
```

Comment:
Should these (the 5 calls in this file) now be `-1` to wait for an unlimited amount of time?